.github/workflows/release-and-split.yml

```diff
@@ -32,10 +32,12 @@ jobs:
         run: pnpm config set store-dir $PNPM_CACHE_FOLDER
       - name: install dependencies
         run: pnpm install
+      - name: make script runnable
+      - run: chmod +x ".github/scripts/update-lockfiles.sh"
```

Comment:
Best guess is that the leading - is the problem. I think this should be:

`run: chmod +x ".github/scripts/update-lockfiles.sh"` (without the - )